justfile

```diff
@@ -352,10 +349,12 @@ f:
     just frontend/run dev
 
 # alias for `pnpm --filter {package} run {script}`
-p package script +args="":
-    pnpm --filter {{ package }} run {{ script }} {{ args }}
+[positional-arguments]
+p package script *args:
```

Comment:
A handful of scripts used `+args=""`, which works, but is technically redundant. `*args` expresses the same thing (optional variadic arguments to gather), and perhaps much more intentionally, especially to relay that they are _optional_, rather than the `+` prefix which indicates at least one is required (a requirement nullified by the default blank that is passed).